nuxt.config.js

```diff
@@ -3,7 +3,16 @@ export default defineNuxtConfig({
     '@nuxt/content'
   ],
   content: {
-    documentDriven: true
+    documentDriven: true,
+    markdown: {
+      remarkPlugins: {
+        'remark-sectionize': {},
+      },
+      toc: {
```

Comment:
What is this doing?

Reply:
@AndyClifford by default only 2 levels are searched for the TOC.

We previously had a structure like:
- h2
    - h3

We now have a structure like: 
- section
    - h2
        - section
            - h3

Therefore we need to go to 4 levels of depth.